config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-presubmits.yaml.tpl

```diff
@@ -266,132 +269,6 @@ presubmits:
-    - org: kubernetes
-      repo: kubernetes
-      base_ref: master
-      path_alias: k8s.io/kubernetes
-    decorate: true
-    always_run: false
-    branches:
-      # The script this job runs is not in all branches.
-      - ^{{ $.branch }}$
-    path_alias: sigs.k8s.io/cluster-api
-    spec:
-      containers:
-        - image: {{ $.config.TestImage }}
-          args:
-            - runner.sh
-            - "./scripts/ci-e2e.sh"
-          env:
-            # enable IPV6 in bootstrap image
-            - name: "DOCKER_IN_DOCKER_IPV6_ENABLED"
-              value: "true"
-            - name: GINKGO_SKIP
-              value: "\\[Conformance\\]"
-          # we need privileged mode in order to do docker in docker
-          securityContext:
-            privileged: true
-          resources:
-            requests:
-              cpu: 3000m
-              memory: 8Gi
-            limits:
-              cpu: 3000m
-              memory: 8Gi
-    annotations:
-      testgrid-dashboards: sig-cluster-lifecycle-cluster-api{{ if eq $.branch "main" | not -}}{{ TrimPrefix $.branch "release" }}{{- end }}
-      testgrid-tab-name: capi-pr-e2e-dualstack-and-ipv6-{{ ReplaceAll $.branch "." "-" }}
-{{ else }}
-  - name: pull-cluster-api-e2e-informing-ipv6-{{ ReplaceAll $.branch "." "-" }}
-    cluster: eks-prow-build-cluster
-    labels:
-      preset-dind-enabled: "true"
-      preset-kind-volume-mounts: "true"
-    extra_refs:
-    - org: kubernetes
-      repo: kubernetes
-      base_ref: master
-      path_alias: k8s.io/kubernetes
-    decorate: true
-    optional: true
-    branches:
-      # The script this job runs is not in all branches.
-      - ^{{ $.branch }}$
-    path_alias: sigs.k8s.io/cluster-api
-    run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|internal|scripts|test|util|webhooks|version)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile)'
-    spec:
-      containers:
-        - image: {{ $.config.TestImage }}
-          args:
-            - runner.sh
-            - "./scripts/ci-e2e.sh"
-          env:
-            # enable IPV6 in bootstrap image
-            - name: "DOCKER_IN_DOCKER_IPV6_ENABLED"
-              value: "true"
-            - name: GINKGO_FOCUS
-              value: "\\[IPv6\\] \\[PR-Informing\\]"
-            - name: IP_FAMILY
-              value: "IPv6"
-          # we need privileged mode in order to do docker in docker
-          securityContext:
-            privileged: true
-          resources:
-            requests:
-              cpu: 3000m
-              memory: 8Gi
-            limits:
-              cpu: 3000m
-              memory: 8Gi
-    annotations:
-      testgrid-dashboards: sig-cluster-lifecycle-cluster-api{{ if eq $.branch "main" | not -}}{{ TrimPrefix $.branch "release" }}{{- end }}
-      testgrid-tab-name: capi-pr-e2e-informing-ipv6-{{ ReplaceAll $.branch "." "-" }}
-{{- end }}
-  - name: pull-cluster-api-e2e-{{ ReplaceAll $.branch "." "-" }}
```

Comment:
Don't we have to keep this one? It is different to `pull-cluster-api-e2e-mink8s-...` 🤔

Reply:
Right - sorry about that.